src/Management/test/Endpoint.Test/Actuators/Hypermedia/HypermediaEndpointTest.cs

```diff
@@ -109,8 +108,8 @@ public async Task Invoke_CloudFoundryDisable_ReturnsExpectedLinks()
         var handler = testContext.GetRequiredService<IActuatorEndpointHandler>();
 
         Links links = await handler.InvokeAsync("http://localhost:5000/foobar", CancellationToken.None);
-        Assert.NotNull(links);
-        Assert.NotNull(links.Entries);
-        Assert.Empty(links.Entries);
+        links.Should().NotBeNull();
+        links.Entries.Should().NotBeNull();
+        links.Entries.Should().BeEmpty();
```

Comment:
```suggestion
        links.Entries.Should().BeEmpty();
```

`links.Entries.Should().BeEmpty()` already checks for not null, so `links.Entries.Should().NotBeNull()` is redundant.